Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -537,5 +537,16 @@ public void GenericTypes()
                 Assert.AreEqual(4.2f, genericTypes.B.Value);
             });
         }
+
+        [Test]
+        public void DeserialiseDefaults()
+        {
+            DeserializeCompileAndLoad<DeserialiseDefaults>("25_DeserialiseDefaults.xml", deserialiseDefaults =>
+            {
+                Assert.AreEqual(Vector2.Zero, deserialiseDefaults.EmptyVector2);
+                Assert.AreEqual(Vector2.Zero, deserialiseDefaults.Empty2Vector2);
+
```

Comment:
Shouldn't need them but I agree, should asset everything.
The main point of the test is if the actual XML can be de-serialised.
But I'll add them now.